garage/tf/launchers/ddpg_pendulum.py

```diff
@@ -0,0 +1,69 @@
+"""
```

Comment:
can you leave a comment with on the launcher with the expected final AverageReturn and the approximate rise time (in iters) to achieve that return?